unyt/equivalencies.py

```diff
@@ -466,29 +473,52 @@ def _convert(self, x, new_dims):
         if new_dims == flux:
             x4 = np.power(x, 4, out=self._get_out(x))
             return np.multiply(
-                pc.stefan_boltzmann_constant_cgs, x4, out=self._get_out(x))
+                pc.stefan_boltzmann_constant_mks, x4, out=self._get_out(x))
         elif new_dims == temperature:
             T4 = np.divide(
-                x, pc.stefan_boltzmann_constant_cgs, out=self._get_out(x))
+                x, pc.stefan_boltzmann_constant_mks, out=self._get_out(x))
             return np.power(T4, 0.25, out=self._get_out(x))
 
     def __str__(self):
         return "effective_temperature: flux <-> temperature"
 
-
 em_conversions = {
-    charge_mks: ("esu", 0.1*speed_of_light_cm_per_s),
-    magnetic_field_mks: ("gauss", 1.0e4),
-    current_mks: ("statA", 0.1*speed_of_light_cm_per_s),
-    electric_potential_mks: ("statV", 1.0e-8*speed_of_light_cm_per_s),
-    resistance_mks: ("statohm", 1.0e9/(speed_of_light_cm_per_s**2)),
-    charge_cgs: ("C", 10.0/speed_of_light_cm_per_s),
-    magnetic_field_cgs: ("T", 1.0e-4),
-    current_cgs: ("A", 10.0/speed_of_light_cm_per_s),
-    electric_potential_cgs: ("V", 1.0e8/speed_of_light_cm_per_s),
-    resistance_cgs: ("ohm", speed_of_light_cm_per_s**2*1.0e-9),
+    "C": ("esu", 0.1*speed_of_light_cm_per_s, "cgs"),
+    "T": ("gauss", 1.0e4, "cgs"),
+    "Wb": ("Mx", 1.0e8, "cgs"),
+    "A": ("statA", 0.1*speed_of_light_cm_per_s, "cgs"),
+    "V": ("statV", 1.0e-8*speed_of_light_cm_per_s, "cgs"),
+    "ohm": ("statohm", 1.0e9/(speed_of_light_cm_per_s**2), "cgs"),
+    "esu": ("C", 10.0/speed_of_light_cm_per_s, "mks"),
+    "Fr": ("C", 10.0/speed_of_light_cm_per_s, "mks"),
+    "statC": ("C", 10.0/speed_of_light_cm_per_s, "mks"),
+    "gauss": ("T", 1.0e-4, "mks"),
+    "G": ("T", 1.0e-4, "mks"),
+    "Mx": ("Wb", 1.0e-8, "mks"),
+    "statA": ("A", 10.0/speed_of_light_cm_per_s, "mks"),
+    "statV": ("V", 1.0e8/speed_of_light_cm_per_s, "mks"),
+    "statohm": ("ohm", speed_of_light_cm_per_s**2*1.0e-9, "mks"),
 }
 
+def _get_em_base_unit(units):
+    unit_str = str(units)
+    if len(unit_str) == 1:
+        return unit_str
+    possible_prefix = unit_str[0]
+    prefix_len = 1
+    if unit_str[:2] == 'da':
+        possible_prefix = 'da'
+        prefix_len += 1
+    if possible_prefix in unit_prefixes:
+        base_unit = unit_str[prefix_len:]
+    else:
+        base_unit = unit_str
+    return base_unit
+
+def check_em_conversion(units):
```

Comment:
this should be prefixed with an underscore so it doesn't end up in the public API

Reply:
Does this function have much overhead? It looks like it gets called during all of the unit conversion functions so optimally it would be very cheap and not add much overhead to those function.